luigi/contrib/redshift.py

```diff
@@ -52,7 +138,7 @@ class RedshiftTarget(postgres.PostgresTarget):
     use_db_timestamps = False
 
 
-class S3CopyToTable(rdbms.CopyToTable):
+class S3CopyToTable(rdbms.CopyToTable, _CredentialsMixin):
```

Comment:
So the issue identified in that SO question seems to be around the Mixin implementing an abstractmethod which is supplied by an earlier inherited class.

Perhaps it's worth swapping the order of inheritance for a future case where `_CredentialsMixin` overrides an abstractmethod which is added to `rdbms.CopyToTable`.

So long as the overall functionality still works, it doesn't hurt anything.

THanks,